### DISCUSSION TITLE
Admin User | Change or Add

### DISCUSSION THREAD

**Discussion by Kuroo2021** [wger-project/wger#1806] [Q&A]
![image](https://github.com/user-attachments/assets/3f50f384-2aaa-4467-9673-5f28d58b6b56)
![image](https://github.com/user-attachments/assets/12808a7b-39d2-4360-85ea-ec3a39e5cc5f)

I can't figure out how to change the password and also the sign in  "username"  for myself. I'm missing something on how to do this? Or just addmyself and give myself full admin access and delete admin.

**Comment by FiveKnives** [comment]
# 🔧 How to Create an Additional Admin User in wger (Docker Setup)

If you're running wger via Docker and want to give another user admin access, follow these simple steps. This grants full admin rights via the built-in `admin` group.

---

## 🐳 Step-by-Step (Docker + Django Shell)

1. **Grant all user roles**

   Grant all roles in the wger Web UI to your user account.

3. **Open the Django shell inside the Docker container:**

   ```bash
   docker compose exec web python3 manage.py shell
   ```

4. **Run the following Python code to promote your user:**

   ```python
   from django.contrib.auth.models import User, Group

   user = User.objects.get(username='your_username')
   user.is_staff = True

   admin_group = Group.objects.get(name='admin')
   admin_group.user_set.add(user)

   user.save()
   ```

   ✅ Replace `'your_username'` with the actual username.

---

## ✅ What This Does

- `is_staff = True` allows the user to log into the Django admin interface.
- Adding them to the `admin` group grants all required permissions in wger.
- `.save()` it at the end

---

## 🧪 Notes

- The `admin` group **must exist** (it’s included by default in wger).
- You do **not** need to set `is_superuser` – the group handles all relevant permissions.
- This works for any user already created via the web UI or scripts.

---

## 💬 Example Use Case

> You created a second user during initial setup and want them to have full admin access without using the default `admin` account.

---

Feel free to copy this into your documentation!
